rule-audit-client/src/components/capiFeed/CapiResults.tsx

```diff
@@ -3,15 +3,15 @@ import AppTypes from "AppTypes";
 import { connect } from "react-redux";
 
 import { selectors as capiSelectors } from "redux/modules/capiContent";
-import {
-  selectors as uiSelectors,
-} from "redux/modules/ui";
+import { selectors as uiSelectors } from "redux/modules/ui";
 import { CapiContentWithMatches } from "services/capi";
 import { notEmpty } from "utils/predicates";
 import CapiFeedItem from "./CapiFeedItem";
 
 type IProps = ReturnType<typeof mapStateToProps>;
 
+const articleNumberFormat = new Intl.NumberFormat("en-GB");
```

Comment:
😍

Reply:
I didn't know about this until I tried to do it!